web/static/js/actions/respondents.js

```diff
@@ -10,34 +11,31 @@ export const RECEIVE_RESPONDENTS_STATS = 'RECEIVE_RESPONDENTS_STATS'
 export const fetchRespondents = (
   projectId,
   surveyId,
-  limit,
-  page,
-  filter = '',
-  sortBy = null,
-  sortAsc = true
+  resetCurrentItems,
```

Comment:
I see no need for this argument. The respondent's reducer has a logic that will reset the items if the action's survey differ from the current store state.

Reply:
OK, thanks for pointing this! I'll try to remove it, while keep everything working 😅